app/models/company.rb

```diff
@@ -70,7 +70,7 @@ def error_if_has_accepted_applications?
 
     membership_applications.reload
 
-    if membership_applications.select(&:is_accepted?).count != 0
+    if membership_applications.select(&:accepted?).count != 0
```

Comment:
I know this is not directly part of this PR, but I think this logic would be preferable:
```
if membership_applications.where(state: 'accepted').any?
```
It avoids instantiating an array and should be faster as a result.